### DISCUSSION TITLE
Do NOT downgrade your chromebook

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#180] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
As stated in the title, don't do it. You guys are losing your minds in desperation, which speaks volumes. You guys don't seem very responsible, and you'll regret your decision most likely. I'm warning you, if you slip up you can get banned from electronics and/or get this on your PERMANANT RECORD. For those who don't understand: this will ruin your chances of getting scholarships, etc, and you most likely will get in huge trouble. From now on, I will delete all downgrading discussions (Will ask 3ch0 of course), so don't make any.

**Comment by Fallenmoon8080** [comment]
How is downgrading your chromebook going to get you in trouble. I've wiped my chromebook of history twice, and have never gotten in trouble. What exactly would you be getting in trouble for? It doesn't delete any policies, and when you're done, just re-upgrade. Please explain what could happen.

**Reply by skoopy-star** [reply]
how wipe history

**Reply by Fallenmoon8080** [reply]
This only works if your admins don't force sync. Steps listed below:
1. Turn off sync
2. Make sure any files you'd like to keep are in google drive.
3. `Esc` + `Refresh` + `Power`
4. Sign in as normal.
5. Enjoy your (hopefully blank history)

Also, If this didn't work, try this  `javascript:var num=prompt("How Times Do You Want This Page To Show Up In your History?%22);done=false;x=window.location.href;for%20(var%20i=1;%20i%3C=num;%20i++){history.pushState(0,%200,%20i==num?x:i.toString());if(i==num){done=true}}if(done===true){alert(%22Flooding%20Successful!\n%20%22+window.location.href+%22%20\nIs%20Now%20In%20Your%20History%20%22+num+(num==1?%22%20time.%22:%22%20Times.%20%22))}`

**Reply by ghost** [reply]
because you're messing with software that you don't own in a way that is not warranted.

**Reply by ghost** [reply]
as I said, I don't CARE if you get caught, it's the fact that most of you will, because (most of) you guys have no idea what you're doing

**Reply by Aka-but-nice** [reply]
> How is downgrading your chromebook going to get you in trouble. I've wiped my chromebook of history twice, and have never gotten in trouble. What exactly would you be getting in trouble for? It doesn't delete any policies, and when you're done, just re-upgrade. Please explain what could happen.

@LucRtheL Yea, he said "downgrading" it doesn't matter if you clear history,  "What exactly would you be getting in trouble for?" bro your taking a chromebook that the school paid money for and gave it to you, then turning it into your own, and yes it does delete polices, you are thinking drunk, get out of the shadow realm, stop playing youself.

**Comment by skoopy-star** [comment]
just downgrading it and using LTBEEF after downgrading it will get me in trouble yes?

**Reply by ghost** [reply]
yes

**Comment by CountBleck** [comment]
If you slip up, then yeah, bad things can happen. If you slip up. It really depends on how well you're being monitored.

**Reply by ghost** [reply]
yeah well don't do it. You will be noticed, and you will screw up. Most of you are like 12 and have absolutely no idea what you're doing

**Reply by CountBleck** [reply]
I won't. I know the risks. Some people are willing to accept the risk. That's the issue. It really depends on how well/strictly you're being monitored. People at my school can most likely get away with it, as long as they don't brick the device. Therefore, it's also a matter of knowing what you're doing, as you said.

**Reply by ghost** [reply]
Yeah but it still violates tons of policies, and GoGuardian can alert teachers when it's not active

**Reply by CountBleck** [reply]
My school rarely uses GoGuardian for monitoring. It usually acts as a filter.

**Reply by ghost** [reply]
It's still a bad thing

**Comment by Theblackmamba824** [comment]
> As stated in the title, don't do it. You guys are losing your minds in desperation, which speaks volumes. You guys don't seem very responsible, and you'll regret your decision most likely. I'm warning you, if you slip up you can get banned from electronics and/or get this on your PERMANANT RECORD. For those who don't understand: this will ruin your chances of getting scholarships, etc, and you most likely will get in huge trouble. From now on, I will delete all downgrading discussions (Will ask 3ch0 of course), so don't make any.

I'm not that desperate i can wait until a fix, It's gonna come sooner or later it'll probably take 2-3 more years

**Comment by 3kh0** [comment]
Tbh, if your school is not that heavy on blocking, you should be good with downgrading or even with a unenroll, for me, when the school year is done, they say oh, it is no enrolled, I just say "yeah it has been acting weird, people keep on doing esc+reload+pwr"
